generated/nidigital/nidigital/session.py

```diff
@@ -3211,7 +3778,7 @@ def reset_device(self):
     def self_calibrate(self):
         r'''self_calibrate
 
```

Comment:
Documentation is missing for `self_test`

Reply:
Maybe this is related to the fact that FancySelfTest is described src/nidigital/metadata/functions.py, rather than functions_addon.py?

Reply:
Hmm.. location shouldn't matter, AFAIK.

From what I see, the doc metadata isn't filled out in https://github.com/ni/nimi-python/blob/f45d1c45d86e476ad7c3b1bbc3e51bf1eb4bd5cc/src/nidigital/metadata/functions.py#L949

Maybe adding there would fix it.

Reply:
I looked at how other APIs, e.g. nidcpower do it, and updated the internal metadata accordingly, which has been reflected in the latest update.